src/frontend/komponenter/Felleskomponenter/Venstremeny/sider.ts

```diff
@@ -0,0 +1,31 @@
+import { BehandlingSteg } from '../../../typer/behandling';
+
+export interface ISide {
+    id: string;
+    navn: string;
+    steg?: BehandlingSteg;
+}
+
+export const sider: ISide[] = [
+    { id: 'SAKSOVERSIKT', navn: 'saksoversikt' },
+    {
+        id: 'VILKÅRSVURDERING',
+        navn: 'vilkårsvurdering',
+        steg: BehandlingSteg.VILKÅRSVURDERING,
+    },
+    { id: 'BEREGNING', navn: 'beregning', steg: BehandlingSteg.VILKÅRSVURDERING },
```

Comment:
Is BEREGNING supposed to be the same steg as VIKÅRSVURDERING?

Reply:
Yes, beregning will not be a seperate step in the future. It will only be a view of the vilkårsvurdering.